Onboarding/intro-and-welcome.md

```diff
@@ -85,13 +85,13 @@ end
 ```stack
 card GoToPrivacyPolicy
      when contact.privacy_policy_accepted == "yes" and
-            (contact.opted_in == "" or contact.opted_in == "no"),
+            contact.opted_in == false,
      then: OptIn do
   log("Privacy Policy accepted and not opted in, go to Opt In")
 end
 
 card GoToPrivacyPolicy
-     when contact.privacy_policy_accepted == "yes" and contact.opted_in == "yes",
```

Comment:
Oof, I noticed that `privacy_policy_accepted` is a Text field rather than a Yes/No, which is very confusing. Probably because I couldn't figure out how to do the assignment / comparisons with Yes/No at the time. Can we change `privacy_policy_accepted` to a Yes/No field as part of this MR please?